### ISSUE TITLE
Expression inside template (template literals) on tags

### ISSUE THREAD

**Issue by rickmed** [gandm/language-babel#221]
I have the following

![image](https://cloud.githubusercontent.com/assets/7104786/17260112/99980012-559c-11e6-8e39-4e53208c8270.png)

See ${link} on href does not get colorized bc the whole thing gets rendered as class=string unquoted html so there is no way to colorize the ${}

**Comment by gandm** [comment]
These templates strings are passed off to [language-mustache](https://github.com/atom/language-mustache) which scopes them. I did it this way to avoid writing a extensions in my grammar to support html strings which atom already has support for. That grammar recurses into the tags so I loose control of the html element/attribute scopes. I do manage to regain control between tags, so that ${name} will have a scope of entity.quail.element.js and correct scopes between.

I only supported this notation because someone wanted angular support.  The only way I could solve this is by writing an html extension to my grammar.
